src/ui/public/timepicker/kbn_global_timepicker.js

```diff
@@ -69,6 +70,7 @@ uiModules
       require: '^kbnTopNav',
       link: ($scope, element, attributes, kbnTopNav) => {
         listenForUpdates($scope);
+        updateGlobalStateWithTime($scope);
 
         setTimefilterValues($scope);
```

Comment:
Call to `setTimefilterValues` is no longer needed since  `updateGlobalStateWithTime` calls `setTimefilterValues`